trulens/nn/quantities.py

```diff
@@ -46,6 +44,13 @@ class QoI(AbstractBaseClass):
     output behavior that the attributions describe.
     """
 
+    def __str__(self):
+        return render_object(self, [])
+
+    # TODO: Need to give a seperate value of y at target instance here since
+    # these are values are interventions. Cannot presently define a QoI that says:
+    # logits of the predicted class for each instance.
```

Comment:
are these comments tied to anything?

Reply:
Yes. https://github.com/truera/trulens/issues/72 and some task on the old task tracker which I cannot find now. Will make a new one and refer to it in the comment.